### ISSUE TITLE
Online validator error

### ISSUE THREAD

**Issue by zyzo** [kewisch/ical.js#157]
Hi,
  The online validator fails with all my `.ics` test files and gives this error : 

```
Error: ReferenceError: ICAL is not defined
Stack: ReferenceError: ICAL is not defined
at validate (http://mozilla-comm.github.io/ical.js/validator.html:37:28)
at HTMLFormElement.onsubmit (http://mozilla-comm.github.io/ical.js/validator.html:80:157)
Error: ReferenceError: ICAL is not defined
Stack: ReferenceError: ICAL is not defined
at validate (http://mozilla-comm.github.io/ical.js/validator.html:49:26)
at HTMLFormElement.onsubmit (http://mozilla-comm.github.io/ical.js/validator.html:80:157)
```

When I choose Decorate checkbox, I got HTTP 405 Not Allowed.

This is an example file with what I tested : https://gist.github.com/zyzo/92063e06a25168ba72dc.

**Comment by kewisch** [comment]
The icalendar looks fine, ran it through http://severinghaus.org/projects/icv/ and after removing the nonstandard CHARSET parameter it passes. Likely something change w.r.t. resource loading which makes the validator not load. I'll take a look.

**Comment by kewisch** [comment]
Ah yes, the validator tries to do: `component = new ICAL.icalcomponent(sourceResult);` which is long outdated. I should probably rewrite it anyway.

**Comment by kewisch** [comment]
Kind of fixed in ddacd8d667d30e68083c9d64df2573f85223696f. The validator is still a large mess codewise, but I think it will at least work now.
